circuits/cpp/barretenberg/cpp/src/barretenberg/honk/proof_system/prover_library.hpp

```diff
@@ -1,10 +1,18 @@
 #pragma once
+#include "barretenberg/common/constexpr_utils.hpp"
 #include "barretenberg/ecc/curves/bn254/fr.hpp"
 #include "barretenberg/plonk/proof_system/proving_key/proving_key.hpp"
-#include "barretenberg/plonk/proof_system/types/program_settings.hpp"
 #include "barretenberg/plonk/proof_system/types/proof.hpp"
-#include "barretenberg/polynomials/polynomial.hpp"
+#include "barretenberg/proof_system/relations/relation_parameters.hpp"
 
+// TODO(@zac-williamson). We used to include `program_settings.hpp` in this file. Needed to remove due to circular
```

Comment:
Please add an issue

Reply:
✅ [#2216](https://github.com/AztecProtocol/aztec-packages/issues/2216)

Reply:
Could you please add the number of the issue in the TODO?